packages/react/src/components/DotcomShell/DotcomShell.js

```diff
@@ -20,7 +20,7 @@ const { prefix } = settings;
  * @param {string} User content
  * @returns {*} DotcomShell component
  */
-const DotcomShell = ({ navigation, children }) => {
+const DotcomShell = ({ navigation, footer, children }) => {
```

Comment:
Maybe we should keep a similar language? `footerType`? Just a thought.

Reply:
@photodow yes good call